src/App.tsx

```diff
@@ -1,27 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { get5First, getAll, getRedGoods } from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const showAllGoods = () => {
+    getAll()
+      .then(good => {
+        setGoods(good);
+      });
+  };
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+  const showOnly5First = () => {
+    get5First()
+      .then(good => setGoods(good));
+  };
 
-    <button type="button" data-cy="red-button">
-      Load red goods
-    </button>
+  const showOnlyRed = () => {
+    getRedGoods()
+      .then(good => setGoods(good));
+  };
 
-    <GoodsList goods={[]} />
-  </div>
-);
+  return (
+    <div className="App">
+      <h1>Dynamic list of Goods</h1>
+
+      <button
+        className="button"
+        type="button"
+        data-cy="all-button"
+        onClick={showAllGoods}
+      >
+        Load all goods
+      </button>
+
+      <button
+        className="button"
+        type="button"
+        data-cy="first-five-button"
+        onClick={showOnly5First}
```

Comment:
Better don't use numbers in function naming
```suggestion
        onClick={showOnlyFiveFirst}
```